source/includes/_portfolio.md

```diff
@@ -17,10 +17,16 @@ After performing a GET request to learn the variable's current value and lock ve
 }
 ```
 
-## Whitelisted Variables
-While in the above file you will find a list of all variables which can be submitted through the portfolio-level endpoints, some of these are prohibited to edit. SCORE_LE_1 is an example of a variable which is not whitelisted as it is used for storing the score of indicator LE1, and therefore requests to alter the value will result in `"error": "Variable could not be found."` being raised with a status `404 not found`.
+## Whitelisted Variables and Variable Patterns
+While in the file referenced above you will find a list of all variables which can be submitted through the portfolio-level endpoints, some of the variables are prohibited to edit. SCORE_LE_1 is an example of a variable which is not whitelisted as it is used for storing the score of indicator LE1, and therefore requests to alter the value will result in `"error": "Variable could not be found."` being raised with a status `404 not found`. This also applies to cases where the user directly submits to a base pattern instead of the actual variable.
```

Comment:
actually removed the `SCORE_...` variables from the whitelisted file this year. Thought to be smart 🤦 

as such this part can be removed since it no longer applies

Reply:
I also removed the mention of whitelist from the section heading